enterprise-suite/es-grafana/akka-exporter.json

```diff
@@ -1,14 +1,138 @@
 [
-  {"graphName":"Mailbox Size", "promQL":["max by (actor)(akka_actor_mailbox_size{ContextTags})"]},
-  {"graphName":"Mailbox Time", "promQL":["max by (actor)(akka_actor_mailbox_time_ns{ContextTags,quantile=\"0.99\"})"]},
-  {"graphName":"Actor Failures Rate", "promQL":["sum by (actor) (irate(akka_actor_actor_failure{ContextTags}[1m]))"]},
-  {"graphName":"Processed Message Rate", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{ContextTags}[1m]))"]},
-  {"graphName":"Processing Time", "promQL":["max by (actor)(akka_actor_processing_time_ns{ContextTags,quantile=\"0.99\"})"]},
-  {"graphName":"Sent Messages Rate", "promQL":["sum by (actor)(irate(akka_actor_sent_messages{ContextTags}[1m]))"]},
-  {"graphName":"Running Actors", "promQL":["sum by (actor)(akka_actor_running_actors{ContextTags})"]},
-  {"graphName":"Unhandled Messages Rate", "promQL":["sum by (actor)(irate(akka_actor_unhandled_message{ContextTags}[1m]))"]},
-  {"graphName":"Dead Letters Rate", "promQL":["sum by (actor) (irate(akka_actor_dead_letter{ContextTags}[1m]))"]},
-  {"graphName":"Dispatcher Queue Size", "promQL":["max by (actor_system) (akka_dispatcher_queue_size{ContextTags})"]},
-  {"graphName":"Dispatcher Pool Size", "promQL":["sum by (actor_system) (akka_dispatcher_pool_size{ContextTags})"]},
-  {"graphName":"Dispatcher Threads", "promQL":["sum by (actor_system) (akka_dispatcher_active_threads{ContextTags})","sum by (actor_system) (akka_dispatcher_running_threads{ContextTags})"]}
+  {
+    "graphName": "Mailbox Size",
+    "promQL": [
+      {
+        "expr": "max by (actor)(akka_actor_mailbox_size{ContextTags,quantile=\"0.99\"})",
```

Comment:
No quantile bit here before.  Was adding it intentional?

Reply:
yes, leaving it out is a bug. max of all quantiles is ... non-sense